bin/tex2stree

```diff
@@ -45,12 +49,14 @@ mjAPI.start();
 mjAPI.typeset({
   math: argv._[0],
   format: "TeX",
-  semantic: true
+  semantic: true,
+  minSTree: argv.min
 }, function (data) {
   if (!data.errors) {
     if (argv.xml) {
-      console.log(data.streeXml);}
+      console.log(argv.min ? data.streeXml : (data.streeXml));}
```

Comment:
shouldn't this just be the original line again, since you are returning `data.streeXML` in both cases?